### ISSUE TITLE
Bots are Disconnecting If I Launch More Than 2 Bot

### ISSUE THREAD

**Issue by microadobe** [GrowtopiaNoobs/Growtopia-Bot#43]
I did implemented ping reply and captcha solving features. It will send ping reply packet after ping request. With that I can let only 2 bots stay in world 24hrs straight(sometimes dc but it will reconnect by itself so np).

When i launch more than 2 bots, lets say 3, They'll connect and They'll'll join the world as always but after like 30 seconds, They'll get "ENET_EVENT_TYPE_DISCONNECT" packet from server and they'll disconnect. After that they cannot log back until I wait like 20mins or changing my IP adress. I don't understand why this is happening. I'm so desperate, what do you think this problem caused from ? 

Note: I did tried multiboxing and started like 15 clients at one pc at the same time. They worked flawlessly. So growtopia is not blocking more than 3 connections per IP.

any help appreciated @GrowtopiaNoobs @DrOreo002 @MoanHawks

**Comment by microadobe** [comment]
any suggestions ? @GrowtopiaNoobs

**Comment by MoanHawks** [comment]
try imagine how much packet you spam with 1 ip

**Comment by microadobe** [comment]
@MoanHawks i mean thanks for your comment. As i said before i tried opening 15accounts on 15 different clients at the same computer and from the same ip adress. They are not disconnecting when they launched from original client. I mean which packet am i missing to send ? Btw i am really open to any paid/free suggestions. Any working free suggestions will definitelly be rewarded either cash or dls.

**Comment by MoanHawks** [comment]
using multiboxing not a spam packet

**Comment by microadobe** [comment]
I dont think this cpp bot that growtopiabots wrote sending too much spam packets but Im am really friendly requesting assistance from you, I am so desperate I tried everything. It is probably very easy piece of code that i cannot examine. Could you help me to modify my code to fix spam packets ? Please. I only want bots to stay online nothing else. I am ready to offer you anyhing. Thank you anyways.

**Comment by MoanHawks** [comment]
https://github.com/GrowtopiaNoobs/Growtopia-Bot/blob/373d101e28870bdb3c9b47a8c5b24c2f24683af0/Growtopia_Bot/userfunc.h#L643

**Comment by microadobe** [comment]
I did a fix to not send this comment more than one time. I'm testing now if that works, I will kindly ask your communication adresses(Discord or Telegram or smthg.) Thank you again. @MoanHawks

**Comment by RefetB** [comment]
@microadobe I have the same issue as you.Did you fixed the problem?If you did can explain?

**Comment by RefetB** [comment]
@MoanHawks I even tried to comment that line.Still getting disconnected.Any ideas?

**Comment by RealGoblins** [comment]
Have anyone found a solution for this issue? 

@microadobe Is there any way I could contact you, such as Discord? Feel free to send me a friend request in Discord: RealGoblins#2300 if you have found a solution.
